arm-sql/2014-04-01/swagger/capabilities.json

```diff
@@ -118,7 +119,9 @@
                 "id": {
                     "readOnly": true,
                     "type": "string",
-                    "description": "The unique ID of the service objective."
+                    "format": "uuid",
```

Comment:
Unfortunately no change is possible. This is what the service is returning, and changing it at this point would be a REST API breaking change. We will have to consider removing this guid from the next api version.